src/Sarif/Autogenerated/Run.cs

```diff
@@ -6,7 +6,9 @@
 using System.Collections.Generic;
 using System.ComponentModel;
 using System.Runtime.Serialization;
```

Comment:
>Serialization [](start = 21, length = 13)

why are we seeing these changes? isn't our dotnet-format exclusion getting honored? we can certainly update the autogenerator code to make this fix.

Reply:
probably when you opened / saved, it separated in groups.
About the real fix, sure that would be a good idea! 

---
In reply to: [510438497](https://github.com/microsoft/sarif-sdk/pull/2125#discussion_r510438497) [](ancestors = 510438497)

Reply:
already fixed

---
In reply to: [510440931](https://github.com/microsoft/sarif-sdk/pull/2125#discussion_r510440931) [](ancestors = 510440931,510438497)